docs/consensus/consensus_chain.md

```diff
@@ -106,6 +106,16 @@ Justifications contain a set of all PoT checkpoints since the parent block up to
 4. DSN sync must be able to terminate early if local chain already contains imported blocks that DSN sync was about to download (doesn’t happen often, but possible)
 5. Node blocks that are finalized and pruned must be much higher than archiving point such that block available through DSN sync and regular Substrate sync have significant overall (5 archived segments worth of blocks right now)
 
+## Fast sync
+
+1. Obtain segment headers from DSN: 1-4 from `sync from DSN implementation`
```

Comment:
"sync from DSN implementation" here should be a link, not code block, right?:
```suggestion
1. Obtain segment headers from DSN (steps 1-4 from [Sync from DSN implementation](#sync-from-dsn-implementation))`
```

And similarly about code blocks below.

Reply:
Sure